analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -51,7 +51,7 @@ class GoDep : PackageManager() {
             // FIXME DRY names of legacy manifest files
             listOf("Gopkg.toml", "glide.yaml", "Godeps.json")
     ) {
-        private val NO_LOCKFILE = "NO_LOCKFILE"
+        private const val NO_LOCKFILE = "NO_LOCKFILE"
 
         private val LEGACY_MANIFESTS = mapOf(
```

Comment:
Should this be `const` too?

Reply:
It cannot, because it's no primitive type.